packages/core/src/integrations/inboundfilters.ts

```diff
@@ -36,198 +35,184 @@ export class InboundFilters implements Integration {
   /**
    * @inheritDoc
    */
-  public setupOnce(): void {
+  public setupOnce(addGlobalEventProcessor: (callback: EventProcessor) => void, getCurrentHub: () => Hub): void {
```

Comment:
```suggestion
  public setupOnce(addGlobalEventProcessor: (processor: EventProcessor) => void, getCurrentHub: () => Hub): void {
```

IMHO this makes it a little clearer that the processor itself is getting passed rather than a callback to run after the processor or something.